tests/functional/test_not_found.py

```diff
@@ -8,6 +8,13 @@
 
 
 @pytest.mark.nondestructive
-def test_is_download_button_displayed(base_url, selenium):
+def test_go_back_button_is_not_displayed_without_history(base_url, selenium):
```

Comment:
Nice job on the tests! These are passing for me locally, but I would recommend you push this to `run-integration-tests` before we merge 👍